### ISSUE TITLE
Using the header/body/footer pattern with no padding leads to spurrious scrollbar

### ISSUE THREAD

**Issue by sturgman** [mdgriffith/elm-ui#68]
The issue can be observed in the following code (https://ellie-app.com/3RhRYSrYTq2a1): 
```
import Browser
import Element exposing (..)
import Html exposing (Html)

type alias Model =
    {}

type Msg
    = NoOp

update : Msg -> Model -> Model
update msg model =
    model

view : Model -> Html Msg
view model =
    layout [] <|
        column [ width fill, height fill ]
            [ row [ width fill ] [ text "header" ]
            , row [ width fill, height fill ] [ text "body" ]
            , row [ width fill ] [ text "footer" ]
            ]

main : Program () Model Msg
main =
    Browser.sandbox
        { init = {}
        , view = view
        , update = update
        }
```

**Expected behavior**
The scrollbar should not be necessary. In fact, the elements are rendered with the right size so that a scrollbar should not appear.

**Diagnosis**
This seems to be related to font rendering and the `line-height` property. The default `line-height` in `elm-ui` is 1, which apparently is too small for most fonts.

**Possible Solution**
@mdgriffith mentioned that perhaps it would be best to increase the default value of `line-height` to 1.2. Also, maybe give the user a way to adjust the default value.

**Versions**
 - OS: Windows 10 HOME 64bit
 - Browser Firefox Dev Edition 64.0b6 (64bit)
 - Elm Version: 0.19
 - Elm UI Version 1.1.0

But this should be observable in any other browsers/OSes

**Comment by netaisllc** [comment]
Yeah, so I duplicated the problem here [ MacOS Chrome Version 70.0.3538.102 (Official Build) (64-bit) ]. 

For the dialect of _OpenSans_ you are using, at _fontSize_ of **20px**, it looks like you'll have to go to a minimum _line-height_ of **1.3** (26px).  That clears the undesired vertical scroll bar in my lab.

Not at all sure the optimal way to set that in the ui package.

HTH.

**Comment by alexkorban** [comment]
Also happens with elm-ui 1.1.5. #bug #has-ellie
